@xen-orchestra/web/src/components/vm/VmVifsTable.vue

```diff
@@ -0,0 +1,278 @@
+                 -->
+                <span v-tooltip class="text-ellipsis name">{{ column.value }}</span>
+              </div>
+              <div v-else-if="column.id === 'ip'" class="ip-addresses">
+                <span class="value text-ellipsis">{{ column.value[0] }}</span>
+                <span v-if="column.value.length > 1" class="typo p3-regular more-ips">
+                  {{ `+${column.value.length - 1}` }}</span
+                >
+              </div>
+              <div v-else v-tooltip="{ placement: 'bottom-end' }" class="text-ellipsis">
+                {{ column.value }}
+              </div>
+            </td>
+          </tr>
+        </template>
+      </VtsDataTable>
+      <UiTopBottomTable :selected-items="0" :total-items="0" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useNetworkStore } from '@/stores/xo-rest-api/network.store'
+import { useVifStore } from '@/stores/xo-rest-api/vif.store.ts'
+import { useVmStore } from '@/stores/xo-rest-api/vm.store.ts'
+import type { XoVif } from '@/types/xo/vif.type.ts'
+import VtsConnectionStatus from '@core/components/connection-status/VtsConnectionStatus.vue'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTableActions from '@core/components/ui/table-actions/UiTableActions.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import useMultiSelect from '@core/composables/table/multi-select.composable'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import {
+  faAlignLeft,
+  faAt,
+  faCaretDown,
+  faEdit,
+  faEllipsis,
+  faHashtag,
+  faPlus,
+  faPowerOff,
+  faTrash,
+} from '@fortawesome/free-solid-svg-icons'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { vifs } = defineProps<{
+  vifs: XoVif[]
+}>()
+
+const { get: getNetwork } = useNetworkStore().subscribe()
+const { get: getVm } = useVmStore().subscribe()
+const { isReady, hasError } = useVifStore().subscribe()
+const { t } = useI18n()
+
+const selectedVifId = useRouteQuery('id')
+
+const getNetworkName = (vif: XoVif) => {
+  const network = getNetwork(vif.$network)!
+
+  return network.name_label ? network.name_label : ''
+}
```

Comment:
I think it would be better to handle the `undefined` case:
```suggestion
const getNetworkName = (vif: XoVif) => {
  const network = getNetwork(vif.$network)

  return network?.name_label ? network.name_label : ''
}
```